library/patient.inc

```diff
@@ -1439,7 +1442,7 @@ function getPatientAge($dobYMD, $nowYMD=null)
         else if ($monthDiff < 0) { $age -= 1; }
     }
     else  {
-        $age = "$ageInMonths month"; 
+        $age = "$ageInMonths " . xl('month');
```

Comment:
This is a numeric value, not the string "month".
Note that the whole getPatientAge function, is very primitive, and should look something like this:

function getPatientAge($dob)
{
$diff= date_diff(date_create($date), date_create($today));
$diff_months =$diff->format('%m')
if ($diff_months <=24)
{
 return "$diff_months" . " " . "xl('months')";
}
else
{
return "$diff->format('%y')" . " " . " xl('years')";
}
}

Reply:
Should be "months" vs. "month" in any case...unless they are less than two months old...which is not a permanent situation.

Reply:
$age would be "15 month" with the word month being translated. I can change it to months

Reply:
This is getting the age in months not years.

Reply:
I was just thinking you could handle both with one function.  In which case, the calling file would hardly need an external function if using date_diff.
Here is the best example of all general use value syntax I have found:
```<?php
$start  = date_create('1988-08-10');
$end 	= date_create(); // Current time and date
$diff  	= date_diff( $start, $end );

echo 'The difference is ';
echo  $diff->y . ' years, ';
echo  $diff->m . ' months, ';
echo  $diff->d . ' days, ';
echo  $diff->h . ' hours, ';
echo  $diff->i . ' minutes, ';
echo  $diff->s . ' seconds';
// Output: The difference is 28 years, 5 months, 19 days, 20 hours, 34 minutes, 36 seconds

echo 'The difference in days : ' . $diff->days;
// Output: The difference in days : 10398```